elastic/kibana
# [ObsUx][Infra-ML] Add parameter to prefill host in the feedback form in hosts view ml flyout

Closes #177700

## Summary

This PR adds a parameter to prefill "Host Anomalies" in the feedback form in hosts view ml flyout (so when the host job is the only option)

## Testing
- Go to Hosts view > Anomaly detection
- Check the feedback form
   - the first question should be prefilled with "Host Anomalies" option
   

https://github.com/elastic/kibana/assets/14139027/217723ab-524e-4fed-9432-79f596eacdbe
